### DISCUSSION TITLE
friends/

### DISCUSSION THREAD

**Discussion by giscus** [Lapis0x0/blog-discussion#3] [Announcements]
# friends/

友链

https://www.lapis.cafe/friends/

**Comment by XenWayne** [comment]
我来辣（
https://xenwayne.top
中途还看你站点失联了XD

这新框架语法有点vue那个模板语法的意思

**Reply by Lapis0x0** [reply]
那会可能是www.lapis.cafe的cdn设置错了（） 好像有好几个星期只能通过lapis.cafe访问

**Comment by MrXnneHang** [comment]
https://xnnehang.top/friends

link: https://xnnehang.top
name: XnneHang's Blog
description: 青石巷道闻星铃，披虹沐雪见本心。他日北寒仙界会，一声道友尽沧桑。
avatar: https://image.baidu.com/search/down?url=https://img3.doubanio.com/view/photo/l/public/p2919322722.webp

**Reply by MrXnneHang** [reply]
or avatar: https://s2.loli.net/2024/06/18/7asx58EjJ6lfwIL.jpg

**Reply by Lapis0x0** [reply]
已添加，很好看的博客主题！😘

**Comment by siberiah2o** [comment]
偶然刷到你的博客，加个友链呗  https://www.dataeast.cn/1729156575996

link: https://www.dataeast.cn/
name: 曲速引擎（Warp Drive）- 小曲博客
description: 专注于计算机方面的博客
avatar: https://www.dataeast.cn/upload/saturn-153576_1280.png

**Reply by Lapis0x0** [reply]
已添加友链，祝好😘

**Comment by Introl-ljl** [comment]
我的博客这几天刚换新架构，内容还在同步ing（。
link: https://blog.introl.top/
name: Introl's Blog
description: 一个OIer的OI弱相关博客
avatar: https://blog.introl.top/_astro/avatar.CTECrO0x_Z2sL1mt.webp

在部署博客的时候搜到了您的博客，参考了很多内容，十分感谢。

**Reply by Lapis0x0** [reply]
已添加友链！博客修改的很有创意😙

**Comment by cloud-oc** [comment]
大佬请加个友链吧Org
link：https://www.cloudofficial.top/
name：Cloud；Blog
description：一个Gamer的博客
avatar: https://avatars.githubusercontent.com/u/137894485

**Reply by Lapis0x0** [reply]
已添加友链！祝好😘

**Comment by cry0404** [comment]
hi，可以和你交换友链吗？前天在你的飞书书架指导下也照着做了一套😂，受到启发很大，而且博主的文章也很有意思，很想和你进一步交流!（不过加 qq 没通过🥺，所以在这边想来申请一下
这是我的站点信息
{
  "站点名称": "cry4o4N0tFound",
  "完整地址": "https://cry4o4n0tfound.cn",
  "Favicon": "https://picture.cry4o4n0tfound.cn/2025/09/b4ed0a1d77cacf879e1b3e8fc2190473.jpg",
  "Description": "Just Study for Fun",
  "RSS": "https://cry4o4n0tfound.cn/atom.xml"
}

**Reply by Lapis0x0** [reply]
友链已添加！还有我这里QQ 没看到有新的好友申请呀，感觉应该是直接被鹅子给过滤掉了🥲
欢迎所有有交流意愿的朋友加我的 QQ：2089931398 来一起交流！
> 好像 giscus 的评论系统在通知上有点小问题，我正在思考要不要迁移到 twikoo

**Reply by cry0404** [reply]
是有点问题，似乎是类似于 issue ？所有在评论区的都会收到消息。评论系统我倒是挺推荐基于 supabase 来设计一个的，这样自定义的同时以后想加东西还可以直接用 supabase，反正我的评论系统实现是基于这个，感觉挺好用😂

**Comment by cloud-oc** [comment]
你好， 我不太记得之前在飞书上弄了什么，感觉你可能找错人了，这是我的Blog链接：https://www.cloudofficial.top/，如果的确是这个，可以再加一下我的QQ：1621454829




1621454829
***@***.***

**Reply by cry0404** [reply]
啊不好意思，我指的是博主，似乎回复到错误位置了🥲

**Comment by LANSGANBS** [comment]
已加友链！博主的内容很有用

link: https://www.lansganbs.cn/
name: LANSGANBS's Blog
description: 红叶最多情，一舞寄相思。
avatar: https://www.lansganbs.cn/images/friends/lansganbs.png

**Reply by Lapis0x0** [reply]
已添加友链😘

**Comment by ZHYxulei** [comment]
已添加友链，我来交友了！
name：大磊の猫窝
link：https://blog.zhyxulei.top
avatar：https://q2.qlogo.cn/headimg_dl?dst_uin=2742291286&spec=5
descr：一个技术宅的自我幻想

**Reply by Lapis0x0** [reply]
你好，感谢先行添加我的链接，很荣幸可以收到友链邀请😘
我看了下你的站点，目前还处在起步阶段（文章未满 10 篇、站龄不足 6 个月）。为保证友链页的内容质量与稳定性，我这边建议的友链申请标准为：满足「建站 ≥6 个月 + 原创 ≥10 篇」。后续持续更新的话非常欢迎再来申请，祝创作顺利。

**Comment by LANSGANBS** [comment]
我的文章是21篇，站龄9月28天。https://www.lansganbs.cn/，您是不是看错网址了...




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Lapis0x0/blog-discussion"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年10月30日(星期四) 下午3:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lapis0x0/blog-discussion] friends/ (Discussion #3)





 
你好，感谢先行添加我的链接，很荣幸可以收到友链邀请😘
 我看了下你的站点，目前还处在起步阶段（文章未满 10 篇、站龄不足 6 个月）。为保证友链页的内容质量与稳定性，我这边建议的友链申请标准为：满足「建站 ≥6 个月 + 原创 ≥10 篇」。后续持续更新的话非常欢迎再来申请，祝创作顺利。

**Reply by Lapis0x0** [reply]
不是不是，这怪 github discussion 的机制，有一个人发信息的话就会通知所有人……

**Reply by XenWayne** [reply]
路人来发送小广播(
giscus的评论通知默认都是抄送所有人哒(
大家被打扰到的可以在github unsubscribe这个discussion XD

**Comment by LANSGANBS** [comment]
已添加友链！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Lapis0x0/blog-discussion"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年10月30日(星期四) 下午2:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lapis0x0/blog-discussion] friends/ (Discussion #3)





 
已添加友链，我来交友了！
 name：大磊の猫窝
 link：https://blog.zhyxulei.top
 avatar：https://q2.qlogo.cn/headimg_dl?dst_uin=2742291286&amp;spec=5
 descr：一个技术宅的自我幻想

**Comment by LANSGANBS** [comment]
原来如此 哈哈 谢谢提醒




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Lapis0x0/blog-discussion"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年10月30日(星期四) 下午3:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lapis0x0/blog-discussion] friends/ (Discussion #3)





 
不是不是，这怪 github discussion 的机制，有一个人发信息的话就会通知所有人……

**Comment by Twoleaf** [comment]
已添加贵站~

站点名称: 夏夜流萤
站点描述: 飞萤之火自无梦的长夜亮起，绽放在终竟的明天。
站点链接: https://blog.cuteleaf.cn
头像链接: https://q1.qlogo.cn/g?b=qq&nk=7618557&s=640

**Reply by Lapis0x0** [reply]
友链已添加！还有，因为本博客目前使用的是 github discussion 作为评论区源，为了避免未来新的友链申请吵到你，建议在github unsubscribe这个discussion🥹
